### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by sonpham2k**
## Related Tickets:
- [#46258](https://edu-redmine.sun-asterisk.vn/issues/46258)

## WHAT this PR does?
Create Migration 

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sonpham2k** [comment]
ready

File: database/migrations/2022_02_26_035356_create_users_table.php [hunk 1]
```diff
@@ -15,11 +15,16 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('user_name');
```
Comment (manhtd-1083) [inline database/migrations/2022_02_26_035356_create_users_table.php:18]:
```suggestion
            $table->string('username');
```

**Review by manhtd-1083 — COMMENTED** [review]
dùng soft delete cho m nhé

**Comment by sonpham2k** [comment]
ready

**Comment by sonpham2k** [comment]
ready

**Pull request merged**